.github/workflows/pr_tests.yml

```diff
@@ -35,6 +35,23 @@ jobs:
 
       - name: Run unit tests
         run: ./gradlew --no-daemon jvmTest
+      - name: Run Detekt (core module)
```

Comment:
Can this run as a separate check than unit tests?

This check says it runs unit tests but also checks detekt which I would like not to mix